File: C#/07 - reverseString.cs
using System;

public class reverseString
{
    public static void Main(string[] args)
    {
        string a = "", b = "";
        int i = 0, ri = 0;
        Console.Write("Enter a string: ");
        a = Console.ReadLine();
        while (i < a.Length)
        {
            ri = a.Length - 1 - i;
            b += a[ri];
            i++;
        }
        Console.WriteLine("reversed string is: " + b);
    }
}

File: C++/07 - reverseString.cpp
#include <iostream>
using namespace std;

int main() {
    string a = "", b = "";
    int i = 0, ri = 0;
    cout << "Enter a string: ";
    cin >> a;
    while (i < a.length()){
        ri = a.length() - 1 - i;
        b += a[ri];
        i++;
    }
    cout << "reversed string is: " << b;
    return 0;
}